packages/wdio-xvfb/src/XvfbManager.ts

```diff
@@ -36,29 +43,34 @@ export interface XvfbOptions {
 const execAsync = promisify(exec)
 
 export class XvfbManager {
-    private force: boolean
-    private packageManagerOverride?: string
-    private forceInstall: boolean
-    private autoInstall: boolean
-    private maxRetries: number
-    private retryDelay: number
-    private log: ReturnType<typeof logger>
+    #force: boolean
+    #packageManagerOverride?: string
+    #forceInstall: boolean
+    #autoInstallSetting: false | true | 'sudo' | { mode?: 'root' | 'sudo', command?: string | string[] }
```

Comment:
autoInstall is cleaner and clearer, let's keep it like that and introduce a new property to control it's mode